app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,64 @@
 // Specific CSS for your home-page
+.banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100vh;
+  top: 0;
+
+}
+
+.titles {
+  color: $gold;
+
+  h1 {
+    font-size: 150px;
+    font-weight: bold;
+    font-family: $title-font;
+    text-shadow: 5px 5px 5px rgb(14, 13, 13);
+  }
+
+  h3 {
+    font-size: 30px;
+    color: white;
+    font-family: $body-font;
+  }
```

Comment:
Is there a reason you change the font to be body-font instead of header-font ?
Also for design cohesion the color should be the same as the rest of the website.
If you want to change it you can change the color in the config/_setup.scss file in the `body` selector but I feel like lightgrey is easier on the eye compared to white in a darker themed website